packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -57,30 +58,56 @@ function getCellStyle(
   displayValue: DisplayValue,
   disableOverflowOnHover = false
 ) {
-  if (field.config.custom?.displayMode === TableCellDisplayMode.ColorText) {
-    return tableStyles.buildCellContainerStyle(displayValue.color, undefined, !disableOverflowOnHover);
+  // How much to darken elements depends upon if we're in dark mode
+  const darkeningFactor = tableStyles.theme.isDark ? 1 : -0.7;
+
+  // See if we're using deprecated settings
+  const usingDeprecatedSettings = field.config.custom.cellOptions.subOptions['color-background'] === undefined;
```

Comment:
Nit: Could we not use `TableCellDisplayMode.ColorBackground` instead of the string literal `color-background` ?

Reply:
You're right, updated this.